datatypes/ring_programs.py

```diff
@@ -1,32 +1,40 @@
-#General imports
+# coding: utf8
+# Copyright 2014-2020 CERN. This software is distributed under the
+# terms of the GNU General Public Licence version 3 (GPL Version 3),
+# copied verbatim in the file LICENCE.md.
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+# General imports
 import numpy as np
 import sys
 import os
 import warnings
 import scipy.constants as cont
 import matplotlib.pyplot as plt
 
-#Common imports
+# Common imports
 from ..devtools import exceptions as excpt
 from ..devtools import assertions as assrt
 from ..utilities import rel_transforms as rt
 from . import blond_function as bf
 from ._core import _function, _expand_function, _check_time_turns,\
-                   _get_dats_types, _check_data_types, _expand_singletons,\
-                   _check_turn_numbers
+    _get_dats_types, _check_data_types, _expand_singletons,\
+    _check_turn_numbers
```

Comment:
Is this indentation the style guide version?  I don't like it...

Reply:
Would this be better? (with all the imports aligned, without or without the parenthesis)
```python
from ._core import (_function, _expand_function, _check_time_turns,
                    _get_dats_types, _check_data_types, _expand_singletons,
                    _check_turn_numbers)
```

Reply:
I find it clearer and a bit more aesthetically pleasing, but it's not exactly a major thing.

Reply:
Corrected